src/client/client4.js

```diff
@@ -1550,6 +1550,13 @@ export default class Client4 {
         );
     };
 
+    getAutocompleteCommandsList = async (teamId, page = 0, perPage = PER_PAGE_DEFAULT) => {
+        return this.doFetch(
+            `${this.getTeamRoute(teamId)}/commands/autocomplete${buildQueryString({page, per_page: perPage})}`,
```

Comment:
This endpoint as it is now won't do anything with the paging but we can leave it there for when the endpoint does the implementation for it